olp-cpp-sdk-authentication/src/IntrospectAppResult.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include "olp/authentication/IntrospectAppResult.h"
```

Comment:
Please swap the blocks.
As per coding style guideline: `Include headers in the following order: Related header, C system headers, C++ standard library headers, other libraries' headers, your project's headers.`